cmd/uplink/cmd/setup.go

```diff
@@ -31,7 +31,7 @@ var (
 	setupCfg    UplinkFlags
 	confDir     string
 	identityDir string
-	isDev       bool
+	defaults    cfgstruct.BindOpt
```

Comment:
maybe it can be just local var in `init`?

Reply:
same here